project-base/app/src/Model/Category/Category.php

```diff
@@ -54,24 +48,6 @@ public function __construct(BaseCategoryData $categoryData)
         $this->akeneoCode = $categoryData->akeneoCode;
     }
 
-    /**
-     * @param \App\Model\Category\CategoryData $categoryData
-     */
-    public function edit(BaseCategoryData $categoryData)
-    {
-        parent::edit($categoryData);
-    }
-
-    /**
-     * @param \App\Model\Category\CategoryData $categoryData
-     */
-    protected function setData(BaseCategoryData $categoryData): void
-    {
-        parent::setData($categoryData);
-
-        $this->svgIcon = $categoryData->svgIcon;
-    }
```

Comment:
We want to have edit and setData method here as in https://github.com/shopsys/shopsys/blob/12.0/project-base/src/Model/Article/Article.php